modules/ducktests/tests/ignitetest/tests/snapshot_test.py

```diff
@@ -63,8 +63,14 @@ def snapshot_test(self, ignite_version):
         loader = IgniteApplicationService(
             self.test_context,
             loader_config,
-            java_class_name="org.apache.ignite.internal.ducktest.tests.snapshot_test.DataLoaderApplication",
-            params={"start": 0, "cacheName": self.CACHE_NAME, "interval": 500_000, "valueSizeKb": 1}
+            java_class_name="org.apache.ignite.internal.ducktest.tests.data_generation.DataGenerationApplication",
+            params={
```

Comment:
Let's have single params dictionary and modify it for second run

Reply:
Not applicable now after adding of the separate DataLoader class.